### PR TITLE
ssh: add comments, nolint rules and rename variable on handlers

### PR DESCRIPTION
**Pull request by henrybarreto**
I've added some comments to improve the readability and remove
error variables, what have not used in more than one place on code.

### CODE REVIEW

File: ssh/server/server.go [hunk 1]
```diff
@@ -25,16 +25,6 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var (
```
Comment (otavio) [inline ssh/server/server.go:28]: I don't like the idea of dropping those are it makes harder to find which errors we return.

**Review by otavio — CHANGES_REQUESTED** [review]

**Comment by henrybarreto** [comment]
> Those errors should be consts.

We cannot create `const` with a function's return because of that, I have used `var` to list errors.

That is the same pattern of API's errors.

**Comment by otavio** [comment]
Great, so we ought to use `var` with the extra errors as well.

File: ssh/server/server.go [hunk 2]
```diff
@@ -72,29 +73,37 @@ func NewServer(opts *Options, tunnel *httptunnel.Tunnel) *Server {
...
 			"session":  glidersession.Context().Value(ssh.ContextKeySessionID),
 		}).Error("Failed to handler the session")
 
+		// finish close the SSH's session.
 		finish := func(session ssh.Session) {
 			if session != nil {
-				session.Close()
+				err := session.Close()
+				if err != nil {
+					log.WithError(err).Error("Failed to finish the session")
```
Comment (otavio) [inline ssh/server/server.go:97]: Here, for example, ought to be in the `var` list.

Comment (henrybarreto) [reply ssh/server/server.go:97]: I thought, in this case, as this is only an error message and not an error itself, let it hardcoded.

The `var` errors are the only errors what the ShellHub's user receives. The messages inside `log.Error` are exclusively for ShellHub logging and specific for one case, and the user does not have contact with it.

Even though, should we put this messages into variables?

Comment (otavio) [reply ssh/server/server.go:97]: Consistency. Put it there.

**Review by gustavosbarreto — DISMISSED** [review]

**Pull request merged**